packages/core/i18n/nls.de.json

```diff
@@ -447,12 +447,24 @@
       "confirmCloseNever": "Niemals bestätigen.",
       "enableCopy": "Aktivieren von ctrl-c (cmd-c unter macOS) zum Kopieren von markiertem Text",
       "enablePaste": "Aktivieren von ctrl-v (cmd-v unter macOS) zum Einfügen aus der Zwischenablage",
+      "profileNew": "Neues Terminal...",
+      "profileDefault": "Standardprofil wählen...",
+      "selectProfile": "Wählen Sie ein Profil für das neue Terminal",
       "shellArgsLinux": "Die Befehlszeilenargumente, die im Linux-Terminal zu verwenden sind.",
       "shellArgsOsx": "Die Befehlszeilenargumente, die im macOS-Terminal zu verwenden sind.",
       "shellArgsWindows": "Die Befehlszeilenargumente, die im Windows-Terminal zu verwenden sind.",
       "shellLinux": "Der Pfad der Shell, die das Terminal unter Linux verwendet (Standard: '{0}'}).",
       "shellOsx": "Der Pfad der Shell, die das Terminal unter macOS verwendet (Standard: '{0}'}).",
       "shellWindows": "Der Pfad der Shell, die das Terminal unter Windows verwendet. (Standard: '{0}').",
+      "shell.deprecated": "Dies ist veraltet, neu können Sie Ihre Shell konfigurieren, indem Sie ein Profil unter 'terminal.integrated.profiles.{0}' anlegen und dessen Namen in 'terminal.integrated.defaultProfile.{0}' als Standard setzen.",
+      "defaultProfile": "Das Standardprofil unter {0}",
+      "profiles": "Die Profile welche zur Erzeugung eines Terminals verwendet werden können. Setzen Sie den Pfad von Hand mit optionalen Parametern.\n\nSezen Sie ein Profile auf `null` um es zu verbergen, z.B.: `{0}: null`.",
+      "profilePath": "Der Pfad der Shell, den dieses Profil benutzt.",
+      "profileSource": "Eine Profilquelle, die Shellpfade automatisch erkennt. Unübliche Installationsorte werden nicht unterstützt und müssen manuell erfasst werden",
+      "profileArgs": "Die Shellparameter, welche dieses Profil verwendet.",
+      "profileEnv": "Ein Objekt mit Umgebungsvariablen die zum Terminalprozess hinzugefügt werden. Setzen Sie Variablen auf `null` um sie aus der Basisumgebung zu löschen",
+      "profileIcon": "Eine codicon ID zur Verwendung mit diesem Terminal.  \nterminal-tmux:\"$(terminal-tmux)\"",
+      "profileColor": "ID einer Terminal-Themenfarbe zur Verwendung mit diesem Terminal.",
```

Comment:
Would you prefer translations by a German speaker or machine translations ;-)? As for the original nls.json: I assume the "defaults" would be taken from the code, right? If so, why do we even have the file?

Reply:
Keeping the german translation in there is reasonable, however you should remove the entries from the `nls.json`. The automatic translation update will not be performed in case the `nls.json` doesn't change after performing the extraction CLI command.

Reply:
Thanks for the insight.